akka-javasdk-maven/akka-javasdk-parent/pom.xml

```diff
@@ -296,6 +296,7 @@
                         </excludes>
                         <systemPropertyVariables>
                             <logback.configurationFile>${logback.configurationFile}</logback.configurationFile>
+                            <akka.javasdk.dev-mode.project-artifact-id>${project.artifactId}</akka.javasdk.dev-mode.project-artifact-id>
```

Comment:
@johanandren Have you also tried with `mvn verify`, running IntegrationTest ?

Reply:
Hmm, but now I feel less confident that the sample actually had any integration test. I'll double check.